packages/tasks/src/model-libraries.ts

```diff
@@ -565,6 +565,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/SandAI-org/MAGI-1",
 		countDownloads: `path:"ckpt/vae/config.json"`,
 	},
+	"magenta-rt": {
+		prettyLabel: "magenta-realtime",
+		repoName: "Magenta RT",
```

Comment:
I think the latest recommendation is to use the same content for both these fields (they might be joined soon).